vagrant/_commons.sh

```diff
@@ -79,10 +79,10 @@ function install_k8s {
         esac
         sudo git clone --depth 1 https://github.com/kubernetes-sigs/kubespray $kubespray_folder -b $kubespray_version
         sudo chown -R "$USER" $kubespray_folder
-        pushd $kubespray_folder
+        pushd "$kubespray_folder"  || exit
```

Comment:
Or would it be simpler to use `set -o errexit`?

Reply:
This change is for fixing a shellcheck warning.

Reply:
I mean that you wouldn't need to handle `cd` errors if you had `errexit` set. `shellcheck` will know that too. But that might of course affect the rest of the script logic (and the scripts using this utility lib).

Reply:
@ipuustin gotcha, I though that `set -o errexit` was declared on that file but it's only declared in the `installer.sh` file. Anyway, I'm going to include it and remove the `exit` instructions.

Reply:
remember to add a note about it in the commit msg too